src/algorithms.jl

```diff
@@ -1,16 +1,17 @@
+const DEFAULT_NLSOLVE_SHOOTING = TrustRegion(; autodiff = Val(false))
+const DEFAULT_NLSOLVE_MIRK = NewtonRaphson(; autodiff = Val(false))
```

Comment:
why autodiff false?

Reply:
I kept it false to ensure closer behavior to the NLsolve default we had previously.

Reply:
With it being a breaking change, it would be best to get it in line with what we do elsewhere and default to forward mode AD